docker/base/Dockerfile

```diff
@@ -48,6 +48,7 @@ RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.n
     man-db \
     lz4-devel \
     jq \
+    diffutils \
```

Comment:
I don't know how you get to e0d4f6e7660c37490e6511f3bef72eeee87f3943 and the previous commits. But if you used `rebase`, you wouldn't get these extra commits?

`git fetch upstream` where upstream points to the current upstream master
`get rebase upstream/master` on your branch where you only have the first commit? Will that do it?

Reply:
that commit you mentioned is mine, which is on top of all previous commits made (specifically that package `diffutils` was added since command `diff` is used in one of the scripts, but CentOS8 image doesn't have `diff` by default)